### ISSUE TITLE
SyncBox Hue Add

### ISSUE THREAD

**Issue by bougyrabbit** [mthyden/PhillipsHueSIMPLSharp#33]
Hello Mthyden,

Thank you very much for your job! Do you have for project to add the SyncBox device?

It should be so nice if you do it :)

Thanks :)

Nadir

**Comment by mthyden** [comment]
Yeah, give me a few days, I don’t have one of those units myself but if I provide you a update I’d appreciate it if you’d test it for me. 

I’m assuming the only support I need to add for it is a “on” “off” for the sync perhaps on the lighting interface module, so the box would show up as essentially another light.

Will that work? Or are there other features aside from on/off?

**Comment by mthyden** [comment]
After investigating this a little further, the Sync Box uses a completely different API then the normal bridge. I would need to get one of these boxes to fully develop control over it. So this will be a larger effort and I have no ETA on when I could start work on it.

I’m currently working on a Crestron Home module which will service a larger community so that is my current priority.

**Comment by bougyrabbit** [comment]
I have a sync box and RMC3 to test if you need. The functions that are
specifically used in this box are to choose the hdmi port (1->4) and sync
the light. If I can help you let me know :)
I understand that you have other priorities :)

Nadir

Le lun. 6 sept. 2021 à 21:09, Monte Thyden ***@***.***> a
écrit :

> After investigating this a little further, the Sync Box uses a completely
> different API then the normal bridge. I would need to get one of these
> boxes to fully develop control over it. So this will be a larger effort and
> I have no ETA on when I could start work on it.
>
> I’m currently working on a Crestron Home module which will service a
> larger community so that is my current priority.
>
